### ISSUE TITLE
This project is abandoned, switch to this one

### ISSUE THREAD

**Issue by madingess** [NathanDuma/LinkedIn-Easy-Apply-Bot#64]
I patched the bot from this repo and made my changes freely available: https://github.com/madingess/EasyApplyBot

**Comment by Briwhee** [comment]
Can you fix the recent changes to questions not being answered due to LinkedIn changing their code?

**Comment by madingess** [comment]
They are fixed in the linked project.

**Comment by Briwhee** [comment]
linkedin changed how questions are read. If you clear your cache and rerun, I think you will run into problems where questions are not being filled with years of experience or the drop down is no longer working.

**Comment by madingess** [comment]
Yes, but these changes are handled by updates to the bot made in [the linked repository](https://github.com/madingess/EasyApplyBot). 

If you use that code and are still experiencing this problem, please open an Issue there.

**Comment by Briwhee** [comment]
Linked in no longer uses classes like fb-form-element-label so that repository doesn't work. It was a change made about two days ago. I tried to raise an issue there, but couldn't find where so I raised it here.

**Comment by madingess** [comment]
I haven't verified the issue yet, but I opened an issue there on your behalf.

https://github.com/madingess/EasyApplyBot/issues/2

**Comment by Briwhee** [comment]
Thanks. I'll respond there. Please try to update chrome/clear your cache to see the issue.

**Comment by madingess** [comment]
I suspect the issues you are experiencing are environment-related. Please see the added instructions there for using the virtual environment. 

If the issue persists or a new issue arises, let's continue using the Issues tab of that repository.

**Comment by Egypt1213** [comment]
Hey are there any working bots for indeed

**Comment by madingess** [comment]
Use the one linked in the post. 
https://github.com/madingess/EasyApplyBot

**Comment by NathanDuma** [comment]
I tested your bot and it didn't fully work. It failed to answer applications with questions. It needs more fixing if you want to maximize your application count.

**Comment by madingess** [comment]
I'm glad to see your alive @NathanDuma . 

If you have any issues with the bot from that repo, please open an issue there.

**Comment by RedmondMann** [comment]
> 

Is anyone working on this project anymore?

**Comment by shinnjr** [comment]
Also curious if this is the most recent version available, not just from Nathan but in the Linkedin Easy Apply space.
